src/main/java/dev/cinema/dao/impl/CinemaHallDaoImpl.java

```diff
@@ -45,7 +45,7 @@ public CinemaHall add(CinemaHall cinemaHall) {
             return session.createQuery(criteriaQuery).getResultList();
         } catch (Exception e) {
             LOGGER.error("Error retrieving all cinema halls", e);
-            throw new RuntimeException();
+            throw new RuntimeException(e);
```

Comment:
```suggestion
            throw new RuntimeException("Error retrieving all cinema halls", e);
```
+ прибери LOGGER.error
Коли ти прокидуєш помилку + логуєш її це йде дублювання інформації